standard.tex

```diff
@@ -26,67 +26,77 @@
+\item All solder joints should be strain relieved with zipties or P clamps.
+\item If you screw up a soldering job, redo the job. When soldering surface mount components, inspect solder joints with a small magnifying glass.
 \item All connections must be tug proof.
+\item All exposed wire should be heatshrunk; all heatshrink should be clear (IREC regulation).
 \item If a connector could be asymmetric, it should be asymmetric, since asymmetric connectors are much harder to plug in backwards.
+\item Use positive fit connectors wherever possible.
 \end{itemize}
-%end of connections subsection -- }}}
+%-- }}}
 
-%start of wiring and assembly subsection -- {{{
+%Wiring and Assembly -- {{{
 \subsection{General Wiring \& Assembly}
 \begin{itemize}
+\item Any electrical system that may be unsafe to approach while powered should have a manual kill switch installed that:
+    \begin{itemize}
+    \item disables all energetics or potentially unsafe components.
+    \item can be operated without requiring the rocket to be taken off the tower (eg. magswitches).
+    \item does not require any software to work.
+    \end{itemize}
 \item Wiring should be done so that the circuit is as readable as possible. If supplies allow, use different a different insulation color for each wire (at the very least, positive should be red, ground should be black, and all other wires should not be red or black).
-\item All wires you use need to be low enough gauge to match the current you're putting through them.
-\item If your wires get handled or moved or torqued at any point, they should be strain relieved on both sides of the torque point.
-\item Strain relieve everything to the point that it can survive violent shaking (if it's actually going in the rocket, it should survive very violent shaking).
-\item Tight wires between two points are accidents waiting to happen. Avoid these.
-\item If you're using an RC lithium polymer battery, it should be fused (those fuckers can sink 50 amps).
+\item All wires used should be rated for the current going through them.
+\item Strain relief:
+    \begin{itemize}
+    \item Wires that get handled, moved, or torqued at any point should be strain relieved on both sides of that point.
+    \item Wires should be relieved to the point where they can survive very violent shaking (especially if the system will be going in the rocket).
+    \item Avoid tight wires between two points.
+    \end{itemize}
+\item RC lithium polymer batteries should be fused (they can sink up to 50 A).
 \item When designing a box, circuit layout, or other assembly, keep in mind that wires and electrical components do take up space. Leave clearance in the design stage; this problem should be solved before fabrication/assembly.
```

Comment:
Would it be worthwhile to include a point about placing hotswappable components in relatively accessible locations if possible?

Reply:
fixed.